pandas/tslib.pyx

```diff
@@ -378,11 +372,47 @@ class Timestamp(_Timestamp):
             value = self.tz_localize(None).value
         else:
             value = self.value
-        result = Timestamp(unit*np.floor(value/unit),unit='ns')
+        result = Timestamp(unit*rounder(value/unit),unit='ns')
         if self.tz is not None:
             result = result.tz_localize(self.tz)
         return result
 
+    def round(self, freq):
+        """
+        return a new Timestamp rounded to this resolution
+
+        Parameters
+        ----------
+        freq : a freq string indicating the rounding resolution
+
+        Notes
+        -----
+        For values exactly halfway between rounded decimal values, 
+        Numpy rounds to the nearest even value. Thus the new Timestamp
```

Comment:
this is not necessary. just put a reference to ``np.round``. This is very clear from the function definition itself.